libparsec/crates/crypto/tests/sequester.rs

```diff
@@ -361,3 +363,17 @@ fn verify_with_different_salt_len() {
         data
     );
 }
+
+#[rstest]
+#[case::empty(b"", CryptoError::Decryption)]
+#[case::only_separator(b":", CryptoError::Algorithm("".into()))]
+#[case::no_signature(b"RSASSA-PSS-SHA256:", CryptoError::DataSize)]
+#[case::signature_too_small(b"RSASSA-PSS-SHA256:\0", CryptoError::DataSize)]
+#[case::missing_separator(b"RSASSA-PSS-SHA256@\0\0", CryptoError::Decryption)]
+fn invalid_signature(#[case] signed: &[u8], #[case] err: CryptoError) {
```

Comment:
Can you add a test for unknown algorithm ?

Reply:
Sure